test/utils/navigation.spec.ts

```diff
@@ -76,6 +77,17 @@ describe("NAVIGATION utils", () => {
     expect(backLink).toEqual(config.UPDATE_FILING_DATE_URL);
   });
 
+  test(`getUpdateReviewStatementBackLink returns the correct URL when not on the Remove journey`, () => {
```

Comment:
Very minor, but I would capitalise the word NOT in the description. Just makes it stand out from the test below.

Reply:
We kept it lowercase to maintain consistency with the other 3 lowercase "not" words in the same file.